components/ContentBadge.tsx

```diff
@@ -0,0 +1,124 @@
+import Badge from "~/components/Badge";
+import { StyleSheet, css } from "aphrodite";
+import colors, { bountyColors } from "~/config/themes/colors";
+import icons, {
+  PostIcon,
+  PaperIcon,
+  HypothesisIcon,
+  QuestionIcon,
+} from "~/config/themes/icons";
+import { useRouter } from "next/router";
+import { breakpoints } from "~/config/themes/screen";
+import { POST_TYPES } from "./TextEditor/config/postTypes";
+import ResearchCoinIcon from "~/components/Icons/ResearchCoinIcon";
+
+const ContentBadge = ({ contentType, label = null, onClick = null }) => {
```

Comment:
This component previously existed in many components. This component should standardize how we render badges specific for content (we also have a generic badge component)